src/components/divider/divider.ts

```diff
@@ -0,0 +1,70 @@
+import { LitElement, html } from 'lit';
+import { property } from 'lit/decorators.js';
+import { themes } from '../../theming/theming-decorator.js';
+import { registerComponent } from '../common/definitions/register.js';
+import { styles } from './themes/divider.base.css.js';
+import { styles as shared } from './themes/shared/divider.common.css.js';
+import { all } from './themes/themes.js';
+
+/**
+ * The igc-divider allows the content author to easily create a horizontal/vertical rule as a break between content to better organize information on a page.
+ *
+ * @element igc-divider
+ *
+ */
```

Comment:
List any CSS variables that the component exposes, for example:

```ts
* @cssproperty --color - The color for the divider line.
```

Looking through the styles I see both `--inset` and `--color`.
@SisIvanova can confirm whether I am correct here.